pkg/rpc/server/server.go

```diff
@@ -694,68 +693,23 @@ func (s *Server) getNEP5Transfers(ps request.Params) (interface{}, *response.Err
 	return bs, nil
 }
 
-func amountToString(amount *big.Int, decimals int64) string {
-	if decimals == 0 {
-		return amount.String()
-	}
-	pow := int64(math.Pow10(int(decimals)))
-	q, r := new(big.Int).DivMod(amount, big.NewInt(pow), new(big.Int))
-	if r.Sign() == 0 {
-		return q.String()
-	}
-	fs := fmt.Sprintf("%%d.%%0%dd", decimals)
-	return fmt.Sprintf(fs, q, r)
-}
-
 // decimals represents decimals value for the contract with the specified scripthash.
 type decimals struct {
```

Comment:
Do we still need this type?

Reply:
No, we don't, just forgot to remove it. Fixed.